pkg/shipclient/client.go

```diff
@@ -1,38 +1,9 @@
 package shipclient
 
 import (
-	channels "github.com/replicatedhq/replicated/gen/go/v1"
-	v1 "github.com/replicatedhq/replicated/gen/go/v1"
 	"github.com/replicatedhq/replicated/pkg/graphql"
-	"github.com/replicatedhq/replicated/pkg/types"
 )
 
-type Client interface {
```

Comment:
I have removed these interfaces because they don't add a ton, and I have some thoughts about how to restructure this who thing. 

I also short-circuited directly to the implementations to improve code nav.